src/connectors/price-ranges/connectPriceRanges.js

```diff
@@ -228,6 +228,60 @@ export default function connectPriceRanges(renderFn, unmountFn) {
 
         return nextState;
       },
+
+      getWidgetState(fullState, { state }) {
```

Comment:
According to the change in `getWidgetSearchParameters`:

- `fullState` should be renamed `uiState`
- `state` should be renamed `searchParam`